habitat-baselines/habitat_baselines/rl/multi_agent/bdp_access_mgr.py

```diff
@@ -52,9 +52,23 @@ def _sample_active_idxs(self):
         num_envs = self._agents[0]._num_envs
         device = self._agents[0]._device
 
-        behav_ids = torch.randint(
-            0, self._pop_config.behavior_latent_dim, (num_envs,), device=device
-        )
+        if self._pop_config.force_all_agents:
+            if num_envs != self._pop_config.behavior_latent_dim:
+                raise ValueError(
+                    f"Must have num_envs={num_envs} equal to behavior latent dim={self._pop_config.behavior_latent_dim}"
```

Comment:
So we can have 1 latent ID per environment to evaluate against all agents in the training population (otherwise I would have to have some sampling interval to iterate through the behavior IDs during eval).